src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -340,11 +340,12 @@ configurationRegistry.registerConfiguration({
 		},
 		'explorer.openEditors.sortOrder': {
 			'type': 'string',
-			'enum': ['editorOrder', 'alphabetical'],
+			'enum': ['editorOrder', 'alphabetical', 'fullPath'],
```

Comment:
I'd rename `alphabetical` to `fileName`, then the new option would be `filePath`. The code would need to be made backwards compatible with the old name.

Reply:
I made this but then reverted changes because editors are not equal to the files. There are other special editor types and generic 'alphabetical' is closer to reality.